lib/Sema/TypeCheckNameLookup.cpp

```diff
@@ -555,13 +555,16 @@ void TypeChecker::performTypoCorrection(DeclContext *DC, DeclRefKind refKind,
 }
 
 static InFlightDiagnostic
-diagnoseTypoCorrection(TypeChecker &tc, DeclNameLoc loc, ValueDecl *decl) {
+diagnoseTypoCorrection(TypeChecker &tc, DeclNameLoc loc, ValueDecl *decl, FunctionRefKind refKind) {
```

Comment:
nit-pick: The style guide for C++ in Swift projects is a 80 character column width. You should format this like so:

```cxx
diagnoseTypoCorrection(TypeChecker &tc,
                       DeclNameLoc loc,
                       ValueDecl *decl,
                       FunctionRefKind refKind) {
```

Same goes for the line of code below this one: you'll need to split it across two lines for it to fit within 80 characters.